community/cloud-foundation/templates/dns_records/dns_records.py

```diff
@@ -11,22 +11,20 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Create DNS record-sets resources for a managed zone"""
+"""This template creates DNS records for a managed zone."""
+
 import string
 import random
 
 
 def generate_config(context):
     """ Entry point for the deployment resources.
-    For each ResourceRecordSet. Create:
-    1. A Change to create it.
-    2. A Change to delete it.
-
-    The name of the action should be unique so create a random
-    string to append to the name field.
+    For each ResourceRecordSet, create a resource with an action to
+    create, and another resource with an action to delete.
+    The action name must be unique. Create a random
+    string and append it to the name field.
```

Comment:
> "The action name must be unique. Create a random string and append it to the name field."  

Not sure if it makes sense as two separate sentences, what about something like:

"The action name must be unique, therefore create a random..."

Reply:
To ensure that the action name is unique, create a random string and append it to the field value.

Reply:
yep ok that sounds good, I'll go ahead with the commit and merge